### ISSUE TITLE
Error loading global state

### ISSUE THREAD

**Issue by jsliacan** [code-ready/crc#179]
When starting `crc` the first time (without `~/.crc` directory peresent), one gets the following error at the `setup` stage.
```
INFO[0000] Error loading global state: open /home/jsliacan/.crc/globalstate.json: no such file or directory 
```
As far as functionality is concerned, everything seems to work fine afterwards.

The steps to replicate:
```
$ rm -rf ~/.crc/
$ crc setup
```
Output:
```
INFO[0000] Error loading global state: open /home/jsliacan/.crc/globalstate.json: no such file or directory 
crc - Local OpenShift 4.x cluster
INFO  Caching oc binary                           
INFO  Copying '/tmp/crc677218342/oc' to '/home/jsliacan/.crc/cache/oc/oc' 
INFO  Setting up virtualization                   
INFO  Setting up KVM                              
INFO  Installing Libvirt                          
INFO  Adding user to libvirt group                
INFO  Enabling libvirt                            
INFO  Starting Libvirt service                    
INFO  Installing crc-driver-libvirt               
INFO  Setting up Libvirt crc network              
INFO  Starting Libvirt crc network                
INFO  Writing Network Manager config for crc      
INFO  Writing dnsmasq config for crc 
```

**Comment by gbraad** [comment]
this happens on every command as the globalstate gets loaded as part of the library's `init`.

**Comment by praveenkumar** [comment]
fixed by #191 since we have those as part of Debug instead info messages.
